raster/identify-raster-cell/README.metadata.json

```
@@ -1,6 +1,6 @@
 {
     "category": "Raster",
-    "description": "\nGet the cell value of a local raster at the clicked location and display the result in a callout.",
+    "description": "Get the cell value of a local raster at the clicked location and display the result in a callout.",
```

this needs to be updated to match the new description 
```suggestion
    "description": "Get the cell value of a local raster and display the result in a callout.",
```

---

Good memory, totally forgot about updating this!